src/site/content/en/lighthouse-accessibility/input-image-alt/index.md

```diff
@@ -34,8 +36,7 @@ in the `alt` text
     Username:
     <input type="text">
   </label>
-  <input type="image" alt="Sign in"
-             src="./sign-in-button.png">
+  <input type="image" alt="Sign in" src="./sign-in-button.png">
 </form>
 ```
 
```

Comment:
Yeah, it makes it sound like it's _less_ important to provide alt text in this case rather than more. Plus, if an image includes _only_ text, it really shouldn't be an image. Cut.